docs/configuration/identity-providers/oidc.md

```diff
@@ -283,6 +300,23 @@ required: yes
 
 The shared secret between Authelia and the application consuming this client. Currently this is stored in plain text.
 
+#### public
+<div markdown="1">
+type: bool
+{: .label .label-config .label-purple }
+default: false
+{: .label .label-config .label-blue }
+required: no
+{: .label .label-config .label-green }
+</div>
+
+The public option enables a client where the end user or a malicious application could reasonably obtain the [client id](#id)
```

Comment:
I would add the following link https://datatracker.ietf.org/doc/html/rfc6749#section-2.1 somewhere around here to help people understand what public refers to in the RFC.

Reply:
Added, thanks for the link!